lib/chef/knife/ssh.rb

```diff
@@ -186,8 +186,8 @@ def get_ssh_attribute(node)
           Chef::Log.debug("Using node attribute #{Chef::Config[:knife][:ssh_attribute]}")
           attribute = Chef::Config[:knife][:ssh_attribute]
         elsif node[:cloud] &&
-              node[:cloud][:public_hostname] &&
-              !node[:cloud][:public_hostname].empty?
+            node[:cloud][:public_hostname] &&
+            !node[:cloud][:public_hostname].empty?
           Chef::Log.debug("Using node attribute 'cloud[:public_hostname]' automatically as the ssh target")
```

Comment:
@danielsdeleo if you look down here, you can see the alignment of code inside the block.  the multiline conditional is being indented another level so its clear that the two lines of code there belong to the conditional above it, and not to the code block inside the conditional.

the 'indent' style means that for multiline operator expressions if you want all the arguments to align you generally need to put the first argument on a line by itself:

```ruby
  raise FooBarException,
    "stuff" <<
    "other stuff" <<
    "yet more stuff"
```

So instead you get:

```ruby
  raise FooBarException,
    "stuff" <<
    "other stuff" <<
    "yet more stuff"
```

but by not using 'indent' what rubocop likes to do instead is:

```ruby
      raise SomeReallyLongExceptionClass, "here's the start of a long comment" <<
                                                                     "and here's the continuation where someone tried to break it to make the lines shorter."
```

and that looks kinda awful to me and I'd prefer:

```ruby
      raise SomeReallyLongExceptionClass, "here's the start of a long comment" <<
        "and here's the continuation where someone tried to break it to make the lines shorter."
```